File: Java/First steps in coding/Exercises/09. Fish Tank/FishTank.java
import java.util.Scanner;

public class FishTank {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        int lengthCm = Integer.parseInt(input.nextLine());
        int widthCm = Integer.parseInt(input.nextLine());
        int heightCm = Integer.parseInt(input.nextLine());
        double spaceTakenPct = Double.parseDouble(input.nextLine());

        double waterSpace = lengthCm * widthCm * heightCm;
        waterSpace -= waterSpace * (spaceTakenPct / 100);
        waterSpace /= 1000;

        System.out.println(waterSpace);
    }
}

File: Python/Basics/First Steps in Coding/Exercises/Fish Tank.py
lengthCM = int(input())
widthCM = int(input())
heightCM = int(input())
pctTaken = float(input()) / 100

tankSpaceCM = lengthCM * widthCM * heightCM; tankSpaceCM -= tankSpaceCM * pctTaken
litersAvailable = tankSpaceCM / 1000

print(litersAvailable)
